reframe/core/environments.py

```diff
@@ -121,7 +121,8 @@ def unload(self):
     def emit_load_commands(self):
         rt = runtime()
         if self.is_loaded:
-            ret = self._load_stmts
+        # FIXME: This is a workaround for issue #423; revisit environment interface
```

Comment:
Please align the comment correctly. Also replace the `revisit [...]` with `the environment interface must be revisited (see issue #456)`